lib/db/mysql.js

```diff
@@ -1326,5 +1326,58 @@ module.exports = function (log, error) {
       })
   }
 
+  const DELETE_RECOVERY_CODES = 'CALL deleteRecoveryCodes_1(?)'
+  const INSERT_RECOVERY_CODE = 'CALL createRecoveryCode_1(?, ?)'
+  MySql.prototype.replaceRecoveryCodes = function (uid, count) {
+
+    // Because of the hashing requirements the process of replacing
```

Comment:
Open to suggestions here. Instead of building out one dynamic insert statement, I choose to insert each recovery code individually. This solution, while less efficient, seemed to be a good compromise on readability and security. 

The stored procedure I used for dynamic insertion, was technically executing an arbitrary sql statement.

After https://github.com/mozilla/fxa-auth-db-mysql/pull/320, I will update this to use `MySql.prototype.writeMultiple`.